index.ts

```diff
@@ -39,7 +39,7 @@ function sleep(seconds: number) {
 }
 
 async function createCertificatePfx() {
-    const base64Certificate = core.getInput('certificate');
+    const base64Certificate = core.getInput('certificate').replace(/(-*)(BEGIN|END)( CERTIFICATE)(-*)/g, '');
```

Comment:
Could we use a `+` instead of ` *` here?

Reply:
Can do!